include/_fbs/DeviceToLocalMessage_generated.h

```diff
@@ -497,6 +455,87 @@ inline ::flatbuffers::Offset<WifiNetwork> CreateWifiNetworkDirect(
       saved);
 }
 
+struct ReadyMessage FLATBUFFERS_FINAL_CLASS : private ::flatbuffers::Table {
+  typedef ReadyMessageBuilder Builder;
+  struct Traits;
+  static FLATBUFFERS_CONSTEXPR_CPP11 const char *GetFullyQualifiedName() {
+    return "OpenShock.Serialization.Local.ReadyMessage";
+  }
+  enum FlatBuffersVTableOffset FLATBUFFERS_VTABLE_UNDERLYING_TYPE {
+    VT_POGGIES = 4,
+    VT_CONNECTED_WIFI = 6,
+    VT_GATEWAY_PAIRED = 8,
+    VT_RFTX_PIN = 10
+  };
+  bool poggies() const {
+    return GetField<uint8_t>(VT_POGGIES, 0) != 0;
+  }
+  const OpenShock::Serialization::Local::WifiNetwork *connected_wifi() const {
+    return GetPointer<const OpenShock::Serialization::Local::WifiNetwork *>(VT_CONNECTED_WIFI);
+  }
+  bool gateway_paired() const {
+    return GetField<uint8_t>(VT_GATEWAY_PAIRED, 0) != 0;
+  }
+  uint8_t rftx_pin() const {
+    return GetField<uint8_t>(VT_RFTX_PIN, 0);
+  }
+  bool Verify(::flatbuffers::Verifier &verifier) const {
+    return VerifyTableStart(verifier) &&
+           VerifyField<uint8_t>(verifier, VT_POGGIES, 1) &&
+           VerifyOffset(verifier, VT_CONNECTED_WIFI) &&
+           verifier.VerifyTable(connected_wifi()) &&
+           VerifyField<uint8_t>(verifier, VT_GATEWAY_PAIRED, 1) &&
+           VerifyField<uint8_t>(verifier, VT_RFTX_PIN, 1) &&
+           verifier.EndTable();
+  }
+};
+
+struct ReadyMessageBuilder {
+  typedef ReadyMessage Table;
+  ::flatbuffers::FlatBufferBuilder &fbb_;
+  ::flatbuffers::uoffset_t start_;
+  void add_poggies(bool poggies) {
+    fbb_.AddElement<uint8_t>(ReadyMessage::VT_POGGIES, static_cast<uint8_t>(poggies), 0);
+  }
+  void add_connected_wifi(::flatbuffers::Offset<OpenShock::Serialization::Local::WifiNetwork> connected_wifi) {
+    fbb_.AddOffset(ReadyMessage::VT_CONNECTED_WIFI, connected_wifi);
+  }
+  void add_gateway_paired(bool gateway_paired) {
+    fbb_.AddElement<uint8_t>(ReadyMessage::VT_GATEWAY_PAIRED, static_cast<uint8_t>(gateway_paired), 0);
+  }
+  void add_rftx_pin(uint8_t rftx_pin) {
+    fbb_.AddElement<uint8_t>(ReadyMessage::VT_RFTX_PIN, rftx_pin, 0);
+  }
+  explicit ReadyMessageBuilder(::flatbuffers::FlatBufferBuilder &_fbb)
+        : fbb_(_fbb) {
+    start_ = fbb_.StartTable();
+  }
+  ::flatbuffers::Offset<ReadyMessage> Finish() {
+    const auto end = fbb_.EndTable(start_);
+    auto o = ::flatbuffers::Offset<ReadyMessage>(end);
+    return o;
+  }
+};
+
+inline ::flatbuffers::Offset<ReadyMessage> CreateReadyMessage(
+    ::flatbuffers::FlatBufferBuilder &_fbb,
+    bool poggies = false,
+    ::flatbuffers::Offset<OpenShock::Serialization::Local::WifiNetwork> connected_wifi = 0,
+    bool gateway_paired = false,
+    uint8_t rftx_pin = 0) {
+  ReadyMessageBuilder builder_(_fbb);
+  builder_.add_connected_wifi(connected_wifi);
+  builder_.add_rftx_pin(rftx_pin);
+  builder_.add_gateway_paired(gateway_paired);
+  builder_.add_poggies(poggies);

```

Comment:
whats poggies nowadays?

Reply:
its a boolean flag i decided to leave in the message just as a "everythings working!" kinda thing, discord has similar with the much more understandable "hello" message